src/test/java/org/gridsuite/modification/server/modifications/DeleteVoltageLevelOnLineTest.java

```diff
@@ -0,0 +1,99 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.modifications;
+
+import com.powsybl.iidm.network.Network;
+import lombok.SneakyThrows;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.NetworkModificationException;
+import org.gridsuite.modification.server.dto.DeleteVoltageLevelOnLineInfos;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.utils.MatcherModificationInfos;
+import org.gridsuite.modification.server.utils.NetworkCreation;
+import org.junit.Test;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.UUID;
+
+import static org.gridsuite.modification.server.NetworkModificationException.Type.DELETE_VOLTAGE_LEVEL_ON_LINE_ERROR;
+import static org.gridsuite.modification.server.utils.MatcherDeleteVoltageLevelOnLineInfos.createMatcherDeleteVoltageLevelOnLineInfos;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author bendaamerahm <ahmed.bendaamer at rte-france.com>
+ */
+public class DeleteVoltageLevelOnLineTest extends AbstractNetworkModificationTest {
+
+    @Override
+    protected Network createNetwork(UUID networkUuid) {
+        return NetworkCreation.createForDeleteVoltageLevelOnLine(networkUuid);
+    }
+
+    @Override
+    protected ModificationInfos buildModification() {
+        return DeleteVoltageLevelOnLineInfos.builder()
+               .type(ModificationType.DELETE_VOLTAGE_LEVEL_ON_LINE)
+               .lineToAttachTo1Id("l1")
+               .lineToAttachTo2Id("l2")
+               .replacingLine1Id("replacementLineId")
+               .replacingLine1Name("replacementLine")
+               .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        return DeleteVoltageLevelOnLineInfos.builder()
+                .type(ModificationType.DELETE_VOLTAGE_LEVEL_ON_LINE)
+                .lineToAttachTo1Id("line00")
+                .lineToAttachTo2Id("line11")
+                .replacingLine1Id("replacingLineId2")
+                .replacingLine1Name("replacingLine2")
+                .build();
+    }
+
+    @Override
+    protected MatcherModificationInfos createMatcher(ModificationInfos modificationInfos) {
+        return createMatcherDeleteVoltageLevelOnLineInfos((DeleteVoltageLevelOnLineInfos) modificationInfos);
+    }
+
+    @Override
+    protected void assertNetworkAfterCreation() {
+        assertNull(getNetwork().getLine("l1"));
+        assertNull(getNetwork().getLine("l2"));
```

Comment:
the modification also deletes a VL, v1 in your case.
assertNull(getNetwork().getVoltageLevel("v1"));